WORKSPACE

```diff
@@ -92,6 +92,33 @@ gazelle_dependencies(go_sdk = "go_sdk")
 load("@io_bazel_rules_go//tests:grpc_repos.bzl", "grpc_dependencies")
 grpc_dependencies()
 
+# ---------------------------------------------------------------------------
+#       Overiride net, text, and sys Go modules to patch new Go update
+# ---------------------------------------------------------------------------
+
+go_repository(
+    name = "org_golang_x_net",
+    importpath = "golang.org/x/net",
+    sum = "h1:hZ/3BUoy5aId7sCpA/Tc5lt8DkFgdVS2onTpJsZ/fl0=",
+    version = "v0.1.0",
```

Comment:
Is there a reason that you picked this version?

Otherwise, I would suggest v0.13.0

Reply:
My apologies for the late reply. I got these versions by inspecting some gazelle file for the version we are using for sys (which was the issue from the start) then found the latest compatible version that matches that version of sys. I can test v0.13.0 and report if it works. If it does, I will change all of them to v0.13.0

Reply:
This still seems to fail for me.

Reply:
That's a good reason to go with the versions you picked. The versions I suggested were the latest before the most obvious breakage, but I agree that there are still problems with them.